src/models/user.ts

```diff
@@ -1,59 +1,60 @@
-import bcrypt from 'bcryptjs'
-import mongoose, { model, Schema } from 'mongoose'
+import bcrypt from 'bcryptjs';
+import mongoose, { model, Schema } from 'mongoose';
 
 mongoose.set('toJSON', {
-    virtuals: true,
-    versionKey: false,
-    transform: function (doc, ret) {
-        delete ret._id
-    },
-})
+  virtuals: true,
+  versionKey: false,
+  transform: function (doc, ret) {
+    delete ret._id;
+  },
+});
 
 const userSchema = new Schema({
-    email: {
-        type: String,
-        required: true,
-        unique: true,
-    },
-    password: {
-        type: String,
-        required: true,
-    },
-    role: {
-        type: String,
-        // type: mongoose.Types.ObjectId,
-        // ref: 'User',
-        default: 'user',
-    },
-    // coordinator: {
-    //   type:mongoose.Types.ObjectId,
-    //   required: false,
-    //   ref: 'User'
-    //  },
-    cohort: {
-        type: mongoose.Types.ObjectId,
-        required: false,
-        ref: 'Cohort',
-    },
-})
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  role: {
+    type: String,
+    // type: mongoose.Types.ObjectId,
+    // ref: 'User',
+    default: 'user',
+  },
+  // coordinator: {
+  //   type:mongoose.Types.ObjectId,
+  //   required: false,
+  //   ref: 'User'
+  //  },
```

Comment:
and this comment can be removed to make clean code.

Reply:
Done.